packages/@sparrow-workspaces/src/features/import-collection/layout/ImportCollection.svelte

```diff
@@ -547,9 +547,7 @@
       on:blur={() => {
         isInputDataTouched = true;
       }}
-      placeholder={isWebApp
-        ? "Example - OpenAPI JSON text"
-        : "Example - OpenAPI JSON text or http://localhost:8080/api-docs"}
+      placeholder={"Example - OpenAPI JSON text or http://localhost:8080/api-docs"}
```

Comment:
@aakashreddy-p  why it has been changed? The isWebApp condition was added intentionally, as we don't have support of localhost import in webapp.

Reply:
yes @Astitva877 , I understand, but they raised the bug because the Figma UI contains the message we added with "localhost." They thought it was an issue with the name and raised a bug for it

Reply:
@aakashreddy-p can you please also attach the placeholder link?

Reply:
https://github.com/actions/runner-images/issues/10636